lib/screens/v2/screen_data.ex

```diff
@@ -484,10 +484,7 @@ defmodule Screens.V2.ScreenData do
     alert_ids =
       instance_map
       |> Map.values()
-      |> Enum.filter(fn widget_struct ->
-        not is_nil(AlertWidgetInstance.impl_for(widget_struct))
-      end)
-      |> Enum.flat_map(&AlertWidgetInstance.alert_ids/1)
+      |> Enum.flat_map(&AlertsWidget.alert_ids/1)
```

Comment:
(Since `AlertsWidget.alert_ids/1` now returns `[]` for types that don't implement the protocol, we no longer need to first filter out non-alert widgets.)